Source/Libraries/openXDA.Model/Emails/DataSourceSetting.cs

```diff
@@ -0,0 +1,37 @@
+﻿//******************************************************************************************************
+//  DataSourceSetting.cs - Gbtc
```

Comment:
Since there is no `DataSourceSetting` table and this is used as a base class, this seems more appropriate as an abstract base class as well.

```suggestion
//  EmailDataSourceSettingBase.cs - Gbtc
```